test/io-tests/test_io.py

```diff
@@ -209,6 +216,23 @@ def test_expected_config_from_environment():
     assert dumpconfig(env=env) == expected
 
 
+def test_expected_config_from_db_settings(defaultenv):
+    "Config should be overriden from database settings"
+
+    config = CONFIGSDIR / "no-defaults.config"
+    env = {
+        **defaultenv,
+        "PGRST_DB_LOAD_GUC_CONFIG": "true",
+    }
+    expected = (
+        (CONFIGSDIR / "expected" / "load-db-guc.config")
```

Comment:
```suggestion
        (CONFIGSDIR / "expected" / "no-defaults-with-db.config")
```

(would need to change the filename, too, of course)